example/src/main/assets/mobile-services.json

```diff
@@ -26,6 +26,12 @@
             "config": {
             }
         }
+    ],
+    "https": [
+        {
+            "host": "https://keycloak-myproject.192.168.37.1.nip.io",
```

Comment:
A hostname doesn't include the protocol in the address, so this might causing issues in performing pinning. Pinning libraries expect the format `keycloak-myproject.192.168.37.1.nip.io`. I think this might be more related to https://github.com/aerogear/mobile-cli/pull/71 maybe?

Reply:
@TommyJ1994 Yep this is an issue with the mobile-cli side, will be pushing up the change along with some other stuff in a little while.